recipes/racket/build.sh

```diff
@@ -0,0 +1,17 @@
+# Start with a minimal build without any packages.
+# Avoid building any packages since they could potentially
+# require a more recent version of the base language than
+# is in the current source directory.
+# Packages can be added once the package manager is configured
+# to work for this particular version number.
+make unix-style PREFIX=$PREFIX PKGS=""
```

Comment:
Please set `CPUS` to be `CPU_COUNT`.

ref: https://github.com/racket/racket/blob/daa7ddeef8c40dd4f1ccd5f158219daf1cecda52/INSTALL.txt#L161

ref: https://github.com/conda-forge/conda-forge-ci-setup-feedstock/blob/b4163ff95b38e6d657a9be3f18caca3db3bb8b20/recipe/run_conda_forge_build_setup_linux#L3-L5

ref: https://github.com/conda-forge/conda-forge-ci-setup-feedstock/blob/b4163ff95b38e6d657a9be3f18caca3db3bb8b20/recipe/run_conda_forge_build_setup_osx#L3-L5